lib/resque/scheduler/delaying_extensions.rb

```diff
@@ -71,9 +71,6 @@ def delayed_push(timestamp, item)
         # First add this item to the list for this timestamp
         redis.rpush("delayed:#{timestamp.to_i}", encode(item))
 
-        # Store the timestamps at with this item occurs
```

Comment:
Do I understand correctly that this set add was only to support the removed functionality?

Reply:
Correct. It was only for a couple of actions on the resque-scheduler web interface that, as far as I know, we never use. If we remove that functionality, we can stop doing all of these Redis set operations and get a noticeable speedup.